docs/docusaurus/docs/bpmn/ch07b-BPMN-Constructs.md

```diff
@@ -3881,6 +3881,24 @@ It is also possible to use [field injection](bpmn/ch07b-BPMN-Constructs.md#field
       <flowable:field name="resultVariable" stringValue="myVar" />
     </flowable:taskListener>
 
+- Since Flowable 6.8.0 scriptable task listeners became a first-class citizen using a more compact syntax, avoiding to hard-code the class name:
```

Comment:
This is a minor nitpick but I dislike adding phrases like "Since Flowable xxx...".   First, the phrase is never removed and it just continues to litter the doc for ages to come and second, the document at main is for the current, latest, and greatest version of the code.   If you are working on a prior release then use the corresponding doc; it is all tagged and available for reference.  Just my opinion.....

Reply:
Ok, could be changed to "Recently".